site/src/docs/components/hero/code.mdx

```diff
@@ -0,0 +1,124 @@
+---
+slug: '/components/hero/code'
+title: 'Hero - Code'
+---
+
+import { Button, Hero, Link } from 'hds-react';
+import ExternalLink from '../../../components/ExternalLink';
+import Playground from '../../../components/Playground';
+import TabsLayout from './tabs.mdx';
+
+export default ({ children, pageContext }) => <TabsLayout pageContext={pageContext}>{children}</TabsLayout>;
+
+## Code
+
+### Code examples
+
+#### Diagonal koros
```

Comment:
There could be more than just one code example. It looks like usually the Code page has the same examples as the variations on the index page.